src/QsCompiler/DataStructures/SyntaxTokens.fs

```diff
@@ -447,7 +448,7 @@ type QsFragmentKind =
     | FunctionDeclaration of CallableDeclaration
     | TypeDefinition of TypeDefinition
     | DeclarationAttribute of QsSymbol * QsExpression
-    | OpenDirective of QsSymbol * QsNullable<QsSymbol>
+    | OpenDirective of QsSymbol * QsSymbol // second symbol will be MissingSymbol if no alias provided
```

Comment:
This is the correct spot that results in a break in IQ# (here https://github.com/microsoft/iqsharp/blob/cad319493cd00934a40e1c37e2856fa18d3a1543/src/Core/Compiler/CompilerService.cs#L119). `ValueOr` exists on `QsNullable` but not on `QsSymbol`. If we follow Sarah's recommendation above, this goes back to `QsNullable` and the break goes away.

Reply:
Wow, I had no idea that fragments were a public API. I can partially revert commit (2), which will simplify this PR as well. I was hoping to get rid of the idea of a null alias entirely but it's not worth breaking things, yeah

Reply:
Done

Reply:
Great, thanks for addressing that! With this change and the change putting back the diagnostics the e2e builds are now passing again 👍